scopes/dependencies/dependency-resolver/dependency-resolver.docs.md

```diff
@@ -1,235 +1,251 @@
-The component configuration will support `"-"` sign as a value for dependency which means to remove it from the final dependency list.
-
-The component configuration will support syntax to remove all dependencies (by type) in case the user wants full control. (syntax is open for discussion)
-for example:
-```js
-"dependencies": {
-   // Remove all dependencies calculated from the workspace
-   "*": "-",
-   // Add custom dependency
-   "teambit.bit/my-awesome-component": "5.5.5"
-   },
+
+### Applying policies on all relevant components
+
+A dependency policy set at the root level of the workspace configuration JSON will affect _all_ components that have the configured package as their dependency (i.e., components that have this module listed in their generated dependency graph).
+**Components that do not have this package as a dependency will not be affected.**
+
+For example:
+
+```json
+// At the root-level of the workspace configuration JSON
+
+// Every component that has 'lodash' as a dependency will use version '3.0.0' of it.
+// This policy will not affect any component that does not have 'lodash' as its dependency.
+
+{
+  "teambit.dependencies/dependency-resolver": {
+    "policy": {
+      "dependencies": {
+        "lodash": "3.0.0"
+      }
+    }
+  }
 }
 ```
 
-for example, consider the following config (co-exist in the same workspace with the workspace configuration described above):
-```js
-"teambit.workspace/variants": {
-    /**
-     * wildcards can be used to configure components under a specific namespace.
-     * this configuration applies the react extensions on all components the `ui` namespace.
-    **/
-    "new-ui/*": {
-      "teambit.dependencies/dependency-resolver": {
-        "dependencies": {
-          "lodash": "-",
-          // will change "teambit.bit/my-awesome-component" version to 5.5.5 for any component that require it
-          "teambit.bit/my-awesome-component": {
-            "version": "5.5.5",
-            "force": false,
-          },
-          // will add "teambit.bit/some-other-package" to all component matching this variant
-          "teambit.bit/some-other-package": {
-            "version": "5.5.5",
-            "force": true,
-          }
-        },
-        "peerDependencies": {
-          "react": ">16.0.1"
+### Applying policies on a selected group of components
+
+Dependency policies can be applied on a specific group of components. This is done using the [`@teambit.workspace/variants`](https://bit.dev/workspace/variants) configuration API.
+
+For example, to set the `1.0.0` version of `classnames` as a dependency of all components located inside the `./components/react` directory (or any of its sub-directories):
+
+```json
+{
+  "teambit.workspace/variants": {
+    "teambit.dependencies/dependency-resolver": {
+      "policy": {
+        "components/react": {
+          "classnames": "1.0.0"
```

Comment:
show an example of using `-` to remove something / or to remove and add it to another lifecycle